### PR TITLE
Foundational systems dashboards

### PR DESCRIPTION
**Pull request by casetone**
check that all namespacing changes by ian are kosher

### CODE REVIEW

File: SSPHP/DCAP/default/data/ui/nav/default.xml [hunk 1]
```diff
@@ -1,27 +1,23 @@
...
 
   <collection label="DfE Digital Services" >
     <a href="ssphp_dfe_service_risk">Service Risk</a>
-    <a href="ssphp_foundational_systems_posture">Posture Dashboard</a>
-    <a href="dummy_dashboard">--------------</a>
-    <a href="ssphp_dfe_service?tkn__service=s194">S194 - CISO Continuous Assurance</a>
-    <a href="dummy_dashboard">--------------</a>
+    <a href="ssphp_dfe_service_posture.xml">Posture Dashboard</a>
+    <collection label="Services" >
+      <a href="ssphp_dfe_service?tkn__service=s194">S194 - CISO Continuous Assurance</a>
```
Comment (akinnane) [inline SSPHP/DCAP/default/data/ui/nav/default.xml:18]: In the future, we should probably generate this from the know list of services.

File: SSPHP/DCAP/default/savedsearches.conf.d/ssphp_use_case_dns.conf.d/ssphp_use_case_dns_cis.conf.d/ssphp_use_case_dns_001.conf.d/ssphp_use_case_dns_001_cis_1-14.conf [hunk 2]
```diff
@@ -77,7 +77,8 @@ search = """
        ssphp.score.threshold.red="-"
 
 | eval ssphp.source.service="DNS",
-       ssphp.source.service_name="DNS"
+       ssphp.source.service_name="DNS",
+       ssphp.use_case.savedsearch="xxssphp_use_case_dns_001_cis_1-14{{environment}}"
```
Comment (akinnane) [inline SSPHP/DCAP/default/savedsearches.conf.d/ssphp_use_case_dns.conf.d/ssphp_use_case_dns_cis.conf.d/ssphp_use_case_dns_001.conf.d/ssphp_use_case_dns_001_cis_1-14.conf:81]: Probably shouldn't start with "xxssphp".

File: SSPHP Documentation/SSPHP_Metrics Documentation/ssphp_help_dashboard_template.xml [hunk 3]
```diff
@@ -5,13 +5,13 @@
...
         <html>
-  ~~~\\SSPHP Documentation\\SSPHP_Metrics Documentation\\S001 - Background.md~~~
+  ~~~\\SSPHP Documentation\\{{app}} Documentation\\S001 - Background.md~~~
         </html>
       </panel>
 
       <panel>
         <html>
-  ~~~\\SSPHP Documentation\\SSPHP_Metrics Documentation\\S004 - Dashboards.md~~~
+  ~~~\\SSPHP Documentation\\{{app}} Documentation\\S004 - Dashboards.md~~~
```
Comment (akinnane) [inline SSPHP Documentation/SSPHP_Metrics Documentation/ssphp_help_dashboard_template.xml:14]: Directory name hasn't been changed in the filesystem.

File: SSPHP/DCAP/default/data/ui/nav/default.xml (hunk 1 above)
Comment (casetone) [reply SSPHP/DCAP/default/data/ui/nav/default.xml:18]: ack

File: SSPHP/DCAP/default/savedsearches.conf.d/ssphp_use_case_dns.conf.d/ssphp_use_case_dns_cis.conf.d/ssphp_use_case_dns_001.conf.d/ssphp_use_case_dns_001_cis_1-14.conf (hunk 2 above)
Comment (casetone) [reply SSPHP/DCAP/default/savedsearches.conf.d/ssphp_use_case_dns.conf.d/ssphp_use_case_dns_cis.conf.d/ssphp_use_case_dns_001.conf.d/ssphp_use_case_dns_001_cis_1-14.conf]: fixed

File: SSPHP Documentation/SSPHP_Metrics Documentation/ssphp_help_dashboard_template.xml (hunk 3 above)
Comment (casetone) [reply SSPHP Documentation/SSPHP_Metrics Documentation/ssphp_help_dashboard_template.xml:14]: relic from my old attempt at using python read|mod|write to template before sam's genius jinja idea

File: SSPHP Documentation/SSPHP_Metrics Saved Searches Autorun Schedules.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-**SSPHP_Metrics Saved Searches Autorun Schedules**
+**{{app}} Saved Searches Autorun Schedules**
```
[the remark this answers is not in the document]
Comment (akinnane) [reply SSPHP Documentation/SSPHP_Metrics Saved Searches Autorun Schedules.md:1]: Docs will be fixed up in a future PR

**Review by akinnane — APPROVED** [review]
LGTM

**Review by pritchyspritch — APPROVED** [review]

**Pull request merged**